pvt/common/hardhat-base-config.ts

```diff
@@ -85,6 +85,21 @@ const contractSettings: ContractSettings = {
     runs: 500,
     viaIR,
   },
+  '@balancer-labs/v3-pool-gyro/contracts/GyroECLPPool.sol': {
```

Comment:
Do we need it?

Reply:
Good catch. Probably an issue when solving conflicts.